datasets/neural_code_search/README.md

```diff
@@ -0,0 +1,158 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-nc-4.0
+multilinguality:
+- monolingual
+size_categories:
+  evaluation_dataset:
+  - n<1K
+  search_corpus:
+  - n>1M
+source_datasets:
+- original
+task_categories:
+- question-answering
+task_ids: []
```

Comment:
could you add a task id here ?
If no task id matches in the tagger you can just put a generic description of the task like
```
task_ids:
- other-my-generic-task-description
```